docs/API.md

```diff
@@ -0,0 +1,100 @@
+# dAppeteer API
```

Comment:
Can we have 1-2 sentences of short introduction what dappeteer is and for what it can be used?